project-base/app/tests/App/Functional/Model/Product/ProductOnCurrentDomainElasticFacadeCountDataTest.php

```diff
@@ -57,6 +57,7 @@ public function testCategory(): void
                 $this->domain->getId(),
```

Comment:
I have just noticed that the first parameter (`$domainId`) in `ProductFilterConfigFactory::createForCategory()` method is not used at all, could you please take a look at it?

Reply:
removed, thanks